react/lib/components/Widget/Widget.tsx

```diff
@@ -115,6 +117,8 @@ interface StyleProps {
   copied: boolean
 }
 
+const DONATION_RATE_STORAGE_KEY = 'paybutton_donation_rate'
```

Comment:
Move this to react/lib/util/constants.ts, otherwise is still just a hardcoded string in a component file.